src/listpack.c

```diff
@@ -540,9 +545,7 @@ lpGetWithSize(unsigned char *p, int64_t *count, unsigned char *intbuf, uint64_t
         if (entry_size) *entry_size = 5 + *count + lpEncodeBacklen(NULL, *count + 5);
         return p + 5;
     } else {
-        uval = 12345678900000000ULL + p[0];
-        negstart = UINT64_MAX;
-        negmax = 0;
+        panic("Invalid listpack encoding");
```

Comment:
Maybe too much to ask, but can we add a panicprintf("Invalid listpack encoding %b", p[0]); ? I would like to have some more information.